python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -65,6 +84,10 @@ def get_secrets_dict(self, secrets_manager) -> Mapping[str, str]:
             {secret["name"]: secret["valueFrom"] for secret in self.secrets},
         )
 
+    def get_environment_dict(self) -> Dict[str, str]:
```

Comment:
Related to my other comment but I see transformation of our env var representation (`List[str]` -> `Dict[str, str]` -> `List[Dict[str,str]]`) to be a bit of an unfortunate complexity caused by our original decision to represent secrets differently in our config than AWS expects them in theirs. And perhaps something we could avoid entirely with env vars.